lib/coins.py

```diff
@@ -545,3 +545,68 @@ class DigiByteTestnet(DigiByte):
     IRC_CHANNEL = "#electrum-dgb"
     RPC_PORT = 15022
     REORG_LIMIT = 2000
+
+
+class FairCoin(Coin):
+    NAME = "FairCoin"
+    SHORTNAME = "FAIR"
+    NET = "mainnet"
+    XPUB_VERBYTES = bytes.fromhex("0488b21e")
+    XPRV_VERBYTES = bytes.fromhex("0488ade4")
+    P2PKH_VERBYTE = 0x5f
+    P2SH_VERBYTE = 0x24
+    WIF_BYTE = 0xdf
+    GENESIS_HASH=('1f701f2b8de1339dc0ec908f3fb6e9b0'
+                  'b870b6f20ba893e120427e42bbc048d7')
+    TX_COUNT = 1000
+    TX_COUNT_HEIGHT = 1000
+    TX_PER_BLOCK = 1
+    IRC_PREFIX = "E_"
+    IRC_CHANNEL = "#fairlectrum"
+    RPC_PORT = 40405
+    PEER_DEFAULT_PORTS = {'t': '51811', 's': '51812'}
+    PEERS = [
+        'fairlectrum.fair-coin.net s',
+        'fairlectrum.fair.to s'
+    ]
+
+    @classmethod
+    def header_offset(cls, height):
+        '''Given a header height return its offset in the headers file.
+        If header sizes change at some point, this is the only code
+        that needs updating.'''
+        return height * 108
+
+    @classmethod
+    def header_len(cls, height):
+        '''Given a header height return its length.'''
+        return 108
```

Comment:
Please remove this method as the `Coin` superclass calculates it from the `header_offset()`.